refarch-integrations/refarch-s3-integration/refarch-s3-integration-starter/src/main/java/de/muenchen/refarch/integration/s3/configuration/S3IntegrationAutoConfiguration.java

```diff
@@ -30,34 +29,38 @@ public class S3IntegrationAutoConfiguration {
 
     @ConditionalOnMissingBean
     @Bean
+    @SuppressWarnings("PMD.CloseResource")
```

Comment:
Why is this suppression needed here? Is it a false positive?

Reply:
Not really it is just not closed here as it is needed for the complete application lifetime.